nil/services/stresser/core/helper.go

```diff
@@ -135,7 +135,7 @@ func (h *Helper) TopUp(addr types.Address, value types.Value) error {
 		return fmt.Errorf("failed to top up via faucet: %w", err)
 	} else {
 		if receipt, err := h.WaitTx(tx); err != nil {
-			return fmt.Errorf("failed to get receipt during top up: %w", err)
+			return fmt.Errorf("failed to get receipt %s during top up: %w", tx, err)
 		} else if !receipt.Success {
```

Comment:
Can you also add this change then?
```suggestion
		} else if !receipt.AllSuccess() {
```